Kickstarter-iOS/ViewModels/RootViewModelTests.swift

```diff
@@ -87,6 +87,108 @@ final class RootViewModelTests: TestCase {
     }
   }
 
+  func testSetBadgeValueAtIndex_AppWillEnterForeground() {
+    let mockApplication = MockApplication()
+    mockApplication.applicationIconBadgeNumber = 100
+
+    self.setBadgeValueAtIndexValue.assertValues([])
+    self.setBadgeValueAtIndexIndex.assertValues([])
+
+    withEnvironment(application: mockApplication) {
+      self.vm.inputs.viewDidLoad()
+
+      self.setBadgeValueAtIndexValue.assertValues(["99+"])
+      self.setBadgeValueAtIndexIndex.assertValues([1])
+    }
+
+    mockApplication.applicationIconBadgeNumber = 50
+
+    withEnvironment(application: mockApplication) {
```

Comment:
Is it necessary to re-set the environment here? 🤔

Reply:
Nope! 👍